src/Features/Core/Portable/ExternalAccess/VSTypeScript/VSTypeScriptTaskListService.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.TaskList;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript;
+
+[ExportLanguageService(typeof(ITaskListService), InternalLanguageNames.TypeScript), Shared]
+internal sealed class VSTypeScriptTaskListService : ITaskListService
```

Comment:
EA impl piece for TS.  This bridges between our internal ITaskListService and the IVSTypeScriptTaskListServiceImplementation api tehy will export.